src/frontends/onnx/frontend/src/op/bitwise_or.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op/bitwise_or.hpp"  
```

Comment:
Please fix code style. You can follow generated suggestions from `Files changed` github tab, or use either `cmake --build ./build --target clang_format_fix_all` to automatically fix formatting or `cmake --build ./build --target clang_format_check_all` to check it locally.